nata/bouncing_balls_animation/main.js

```diff
@@ -0,0 +1,102 @@
+const canvas = document.querySelector("canvas");
+const ctx = canvas.getContext("2d");
+const width = (canvas.width = window.innerWidth);
+const height = (canvas.height = window.innerHeight);
+
+function random(min, max) {
+  const num = Math.floor(Math.random() * (max - min + 1)) + min;
+  return num;
+}
```

Comment:
You can simplify it:

```suggestion
function random(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min;
}
```